scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -448,6 +449,44 @@ TaskResult getTaskResult(String jobId, String taskName) throws NotConnectedExcep
             UnknownJobException, UnknownTaskException, PermissionException;
 
     /**
+     * Get the results for a set of tasks in the given jobId and filtered by a given tag.
+     * A user can only get HIS result back.<br>
+     * If the job does not exist, an UnknownJobException is sent with the proper message.<br>
+     * So, if you have the right to get the task result that is in the job represented by the given jobId and if the job and task name exist,
+     * so you will receive the result. In any other cases a schedulerException will be thrown.<br>
+     *
+     * @param jobId the job in which the task result is.
+     * @param taskTag the tag used to filter the tasks in which the result is.
+     * @return a job Result containing information about the result.
+     * 		If the task result is not yet available, null is returned.
+     * @throws NotConnectedException if you are not authenticated.
+     * @throws UnknownJobException if the job does not exist.
+     * @throws PermissionException if you can't access to this particular job.
+     */
+    List<TaskResult> getTaskResultByTag(JobId jobId, String taskTag) throws NotConnectedException,
```

Comment:
I would say getTaskResultsByTag as in comparison with other getTaskResult methods, this one returns a List